release-notes.md

```diff
@@ -27,6 +27,16 @@ This document contains release notes per zib, indicating differences with their
 * Removed the role extension (http://fhir.nl/fhir/StructureDefinition/nl-core-relatedperson-role) because RelatedPerson.relationship has changed from 0..1 to 0..1. The zib concept role is now mapped to a slice on relationship.
 * Updated to new zib-NameInformation and zib-AddressInformation and zib-ContactInformation profiles.
 
+## zib-FunctionalOrMentalStatus
+* SNOMED codes functionalCode and mentalCode on `Observation.category` have been removed
+* The value lists StatusNameICFCodelist, StatusNameNOCCodelist, StatusNameOMAHACodelist, StatusNameSnomedCodelist are combined into one list and modulated on `Observation.code`
+* The value lists StatusValueICFCodelist, StatusValueNOCCodelist, StatusValueOMAHACodelist, StatusValueSnomedCodelist are combined into on list and modulated on `Observation.value[x]:valueCodeableConcept`
+* `EffectiveDateTime` is modulated on its own slice
+* The comment element is modulated on `Observation.note.text` instead of `Observation.comment`
+* MedicalDevice extension have been removed see comment on root element for the explanation
+* The patient reference has been moved to the nl-core layer
+
```

Comment:
Ik vind de release notes wel heel erg gedetailleerd. Kan hoor (wie ben ik), maar volgens mij hoeven hier alleen de wijzigingen die erg opvallend zijn of die niet helemaal volgens de guidelines zijn. Alles wat 'logisch' is, hoeft hier niet volgens mij.

Also: je gebruikt vaak 'modulated' - maar dat klopt niet. Bedoel je 'modelled'? Of 'mapped'?